cmd/acr/purge.go

```diff
@@ -48,15 +50,18 @@ const (
   - Delete all tags older than 1 day in the example.azurecr.io registry inside the hello-world repository, with 4 purge tasks running concurrently
 	acr purge -r example --filter "hello-world:.*" --ago 1d --concurrency 4
 	`
-	maxPoolSize              = 32 // The max number of parallel delete requests recommended by ACR server
-	manifestListContentType  = "application/vnd.docker.distribution.manifest.list.v2+json"
-	manifestIndexContentType = "application/vnd.oci.image.index.v1+json"
-	linkHeader               = "Link"
+	maxPoolSize                 = 32 // The max number of parallel delete requests recommended by ACR server
+	mediaTypeDockerManifestList = "application/vnd.docker.distribution.manifest.list.v2+json"
+	mediaTypeArtifactManifest   = "application/vnd.oci.artifact.manifest.v1+json"
+	mediaTypeImageManifest      = v1.MediaTypeImageManifest
+	mediaTypeImageIndex         = v1.MediaTypeImageIndex
+	headerLink                  = "Link"
 )
 
 var (
 	defaultPoolSize        = runtime.GOMAXPROCS(0)
 	concurrencyDescription = fmt.Sprintf("Number of concurrent purge tasks. Range: [1 - %d]", maxPoolSize)
+	manifestMediaTypes     = set.Set[string]{mediaTypeDockerManifestList: {}, mediaTypeArtifactManifest: {}, mediaTypeImageManifest: {}, mediaTypeImageIndex: {}}
```

Comment:
Apparently, you need a new constructor for `Set`. See `set.go` for more comments.